zjit/src/hir.rs

```diff
@@ -566,6 +566,9 @@ pub enum Insn {
 
     /// Side-exit into the interpreter.
     SideExit { state: InsnId, reason: SideExitReason },
+
+    /// Increment a counter in ZJIT stats
+    IncrCounter(Counter),
```

Comment:
Can you make this Display nicely?

Reply:
I thought `IncrCounter(zjit_insns_count)` was good enough but I guess you don't like parens. Fixed in https://github.com/ruby/ruby/pull/14034/commits/5015f69b5e9b977d03baa2c95d8d636754b8c932